src/terrainEditor/LevelTerrainEditorModal.js

```diff
@@ -171,10 +165,10 @@ export default function LevelTerrainEditorModal(props: Props): React$Node {
 		});
 
 		canvas.toBlob((blob) => {
-			downloadBlob(
-				blob,
-				`Level terrain for ${currentCoordinates[0]}, ${currentCoordinates[1]}, ${currentCoordinates[2]}.png`
-			);
+			fileSave(blob, {
+				fileName: `Level_Geometry_${currentCoordinates[0]}_${currentCoordinates[1]}_${currentCoordinates[2]}`,
```

Comment:
This file name is for human eyes, feel free to use spaces
```suggestion
				fileName: `Level terrain for ${currentCoordinates[0]}, ${currentCoordinates[1]}, ${currentCoordinates[2]}`,
```